pyro/infer/autoguide/guides.py

```diff
@@ -588,19 +588,21 @@ def get_posterior(self, *args, **kwargs):
         Returns a LowRankMultivariateNormal posterior distribution.
         """
         loc = pyro.param("{}_loc".format(self.prefix), self._init_loc)
+        scale = pyro.param("{}_scale".format(self.prefix),
+                           lambda: loc.new_full((self.latent_dim,), self._init_scale),
+                           constraint=constraints.positive)
         factor = pyro.param("{}_cov_factor".format(self.prefix),
                             lambda: loc.new_empty(self.latent_dim, self.rank).normal_(
-                                0, self._init_scale * (0.5 / self.rank) ** 0.5))
-        diagonal = pyro.param("{}_cov_diag".format(self.prefix),
-                              lambda: loc.new_full((self.latent_dim,), 0.5 * self._init_scale ** 2),
-                              constraint=constraints.positive)
+                                0, self._init_scale / self.rank ** 0.5))
```

Comment:
I think you want `.normal_(0, 1 / self.rank ** 0.5)` here.

Reply:
done.